build

```diff
@@ -1272,6 +1287,10 @@ for RECIPEFILE in "${RECIPEFILES[@]}" ; do
     if test -n "$VM_TYPE" -a -z "$RUNNING_IN_VM" ; then
 	vm_first_stage
 	cleanup_and_exit
+    else
+        # chroot based builds
+        # rsync as source and dest could be same
+        test -n "$CCACHE" && rsync -v "$CCACHE" "$BUILD_ROOT/.build.oldpackages/_ccache.tar.gz"
```

Comment:
Hmm, doesn't copy_oldpackages delete this if --oldpackages is not used?

Reply:
yes, I will move the rsync call below the copy_oldpackage call